### DISCUSSION TITLE
surilogcompress

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#3844] [Unsupported Versions]
I get an hourly email from the my system from /usr/local/bin/surilogcompress.

The body just has a few garbage characters in it.

Obviously, this is the results from a cron.hourly job, which from the sound of the name is just a log compression script for suricata.

They question I have is, "Should I be getting this?  Should it be readable?  Should it really be /dev/null instead?

This is 2.3.40 on Ubuntu.

Thanks for the help.

**Comment by TOoSmOotH** [comment]
I put in #3917 to address this.

**Reply by DebianGuru** [reply]
Thanks.
